scripts/freeze.sh

```diff
@@ -1,4 +1,4 @@
-for file in $(find . -name '*.dhall' -print | perl -n -e '$x = $_; $x =~ tr%/%%cd; print length($x), " $_";' | sort -rn -k 1 | awk '{print $2}'); do
+for file in $(find . -path ./src -prune -o -name '*.dhall' -print | perl -n -e '$x = $_; $x =~ tr%/%%cd; print length($x), " $_";' | sort -rn -k 1 | awk '{print $2}'); do
```

Comment:
we may be able to replace this with a `dhall resolve --transitive-dependencies` in the src repo now that the cache doesn't interfere.  I'll co the branch and have a go.

Reply:
dangit at first blush looks like the resolver doesn't keep the order, i.e. the files listed at top of output have dependencies in middle somewhere :exploding_head: